File: 프로그래머스/0/181909. 접미사 배열/접미사 배열.py
def solution(my_string):
    answer = []
    
    for i in range(len(my_string)):
        answer.append(my_string[i:len(my_string)])
        
    answer.sort()
    # 위 코드 지우고
        
    return answer
    # return sorted(answer)를 사용해도 됨

File: 프로그래머스/0/181909. 접미사 배열/접미사 배열.java
import java.util.*;

class Solution {
    public String[] solution(String my_string) {
        ArrayList<String> lists = new ArrayList<>();
        
        for(int i=0; i<my_string.length(); i++){
            String cut = my_string.substring(i, my_string.length());
            lists.add(cut);
        }
        
        Collections.sort(lists); // 사전순으로 정렬
        
        return lists.toArray(new String[0]);
    }
}